device-selection-explainer.md

```diff
@@ -207,13 +289,19 @@ const support = await context.querySupport({
 console.log(support); // "optimized" or "fallback"
 ```
 
-The next phase in developing device selection is therefore to explore this proposal and eventually others.
+The next phase in developing device selection is, therefore, to explore this proposal and eventually others.
+
+Other use cases were also raised in [this comment](https://github.com/webmachinelearning/webnn/issues/815#issuecomment-2658627753) for real-time video processing:
 
-Other use cases were raised as well, in [this comment](https://github.com/webmachinelearning/webnn/issues/815#issuecomment-2658627753) for realtime video processing:
+> 1. If the user selects functionality like background blur, we want to offer the best quality the device can offer. So, the product has a small set of candidate models and technologies (WebNN, WebGPU, WASM) that it has to choose between. Accelerated technologies come with an allowance for beefier models.
 
-> 1. If the user selects to use functionality like background blur, we want to offer the best quality the device can offer. So the product has a small set of candidate models and technologies (WebNN, WebGPU, WASM) that it has to choose between. Accelerated technologies come with allowance for beefier models.
+> 2. The model/tech chooser algorithm needs to be fast, and we need to avoid spending seconds or even hundreds of milliseconds to figure out if a given model should be able to run accelerated. So, for example, downloading the entirety (could be large), compiling, and try-running a model seems infeasible.
 
-> 2. The model/tech choser algorithm needs to be fast, and we need to avoid spending seconds or even hundreds of milliseconds to figure out if a given model should be able to run accelerated. So for example downloading the entirety (could be large things..), compiling & try-running a model seems infeasible.
+Given the discussion in Issue #815 ([comment](https://github.com/webmachinelearning/webnn/issues/815#issuecomment-2635299222), [comment](https://github.com/webmachinelearning/webnn/issues/815#issuecomment-2638389869)), the developer use case (for frameworks, not for websites) seems to be:
+- Before downloading/loading a model, the developer wants to know if, e.g., the GPU can be used for inference with WebNN.
```

Comment:
This feedback https://github.com/webmachinelearning/webnn/issues/815#issuecomment-2995936461 from Google Meet suggests it is beneficial to know before downloading the model if WebNN _cannot_ be used for inference adhering to the `MLPowerPreference` (if any).

This seems like a new use case, and is different from the currently documented "GPU can be used for inference with WebNN" signal.

Reply:
I captured this in a more generic requirement in this [commit](https://github.com/webmachinelearning/webnn/pull/860/commits/6c6b0fc7944091a075a25e79fd9d9195d2734fac).